src/views/Bridge.vue

```diff
@@ -372,7 +385,10 @@ export default class Bridge extends Mixins(
   }
 
   get isMaxAvailable(): boolean {
-    if (!this.areNetworksConnected || !this.isRegisteredAsset || !this.asset) {
+    if (!(this.areNetworksConnected && this.asset)) {
+      return false;
+    }
+    if (!this.isRegisteredAsset && !this.isSoraToEvm) {
```

Comment:
`!this.isRegisteredAsset && !this.isSoraToEvm` -> `!(this.isRegisteredAsset || this.isSoraToEvm)`